README.txt
Added logo and animation text for out title. 


changelog.txt
- ADDED LOGO


PIZZA.sh
#!/bin/bash

clear
tput setaf 6
tput setab 1
tput bold
cat LOGO.txt | randtype -l
sleep 1s

#---------To start your order
tput sgr0
tput bold
tput setaf 1
read -p "Hello, welcome to Generation Pizza! Would you like to start your order? " answer1

if [ "$answer1" = "yes" ] || [ "$answer1" = "Yes" ] || [ "$answer1" = "y" ] || [ "$answer1" = "Y" ]
then 
tput bold
tput setaf 3
	printf "\nHere are the daily specials!\n" | randtype -t 0,10000
	printf "\nChoose any 2 or more for 10.99 each!\n" | randtype -t 0,10000 
	printf "\nAll pizzas, 3 toppings for 7.99!\n" | randtype -t 0,10000
sleep 1s
#------------Pizza Options
printf "\n\nHere are our menu options!\n" | randtype -t 0,10000
printf "\nSize options:\n" | randtype -t 0,10000
printf "\n1.) Extra-Large (20 inches)-18.99"| randtype -t 0,10000
printf "\n2.) Large (16 inches)-15.99" | randtype -t 0,10000
printf "\n3.) Medium (13 inches)-13.99" | randtype -t 0,10000
printf "\n4.) Small (10 inches)-10.99" | randtype -t 0,10000
printf "\n5.) Individual portion (8 inches)-8.99\n" | randtype -t 0,10000
sleep 1s
#------------Styles
printf "\nSpecialty Pizzas:\n" | randtype -t 0,10000
printf "\n- Pepperoni" | randtype -t 0,10000
printf "\n- 5 Cheeses" | randtype -t 0,10000
printf "\n- Hawaiian" | randtype -t 0,10000
printf "\n- Vegan Veggie Lovers" | randtype -t 0,10000
printf "\n- Meat Lovers"| randtype -t 0,10000
printf "\n- Philly Cheese" | randtype -t 0,10000
printf "\n- Buffalo Chicken\n" | randtype -t 0,10000
sleep 1s
#--------------Additional Toppings
printf "\nAdditional Toppings 1.50 extra/each:\n" | randtype -t 0,10000
printf "\n- Ham"| randtype -t 0,10000
printf "\n- Vegan Beef" | randtype -t 0,10000
printf "\n- Chicken" | randtype -t 0,10000
printf "\n- Mushrooms" | randtype -t 0,10000
printf "\n- Jalapenos Peppers" | randtype -t 0,10000
printf "\n- Black Olives"| randtype -t 0,10000
printf "\n- Green Peppers" | randtype -t 0,10000
printf "\n- Sun Dried Tomatoes" | randtype -t 0,10000
printf "\n- Onions" | randtype -t 0,10000
printf "\n- Pineapple" | randtype -t 0,10000
sleep 1s


 
sleep 1
tput setaf 1

printf "\n\n===============================\n\n"

read -p "To start off, how many pizzas are you getting today? " n2
 
#-------------Loop for amount of pizzas

for x in $(seq $n2)
do 
  echo " "
  echo "*****************Pizza # $x*********************"
  read -p "What size would you like your Pizza to be? " s3
    sizeArray[$x]=$s3
sleep 1

#-------------Style of pizza  

  read -p "Which Specialty Pizza would you like? " s4
     specialtyArray[$x]=$s4
sleep 1

#----------------Additional Toppings

   read -p "Would you like to add any additional toppings? " a5
 
       if [ "$a5" = "yes" ] || [ "$a5" = "Yes" ] || [ "$a5" = "y" ]
          then
            read -p "List your toppings " t7
                  toppingArray[$x]=$t7
          elif [ "$answer5" = "*" ]
            then 
             break 
               else   
        printf "\n\n~~~~~~~~~~~~~~~~~~~~~~Alright lets continue!~~~~~~~~~~~~~~~~~~~~~~~\n\n"

        fi
done

sleep 1

#-------------Listing Order

for x in ${!sizeArray[*]}
do 
 printf "\nPizza #$x: "
printf "\nSize:${sizeArray[$x]}\nSpecialty:${specialtyArray[$x]}\nToppings:${toppingArray[$x]}\n\n" 
done

else  
        printf "\n\nSee you again!"
sleep 2
clear
fi

 


DPIZZA.sh
#!/bin/bash

#------------TITLE LOGO
clear
tput setaf 6
tput setab 1
tput bold
cat LOGO.txt | randtype -l
sleep 1s
tput sgr0

#------ARRAYS
pizzaArray=("Cheese" "Pepperoni" "Hawaiian" "Veggie" "Vegan" "MeatLovers")
sizeArray=("Extra Large" "Large" "Medium" "Small")
priceArray=('$15.99' '$13.99' '$10.99' '$8.99')
<<<<<<< HEAD
tPizzaArray=()
tSizeArray=()
=======
tPizzaArray=$($PS3)
counter=0
>>>>>>> 76695d3fe2856ab497a2c352fbe8fda06ffc5335
#---------Greeting
tput setaf 2
tput bold
printf "\n\nWelcome to PIZZA GENERATION!\n" | randtype -t 0,10000
tput sgr0
sleep 1s
#------------------------------=======----------PIZZA MENU
tput setaf 3
tput bold
printf "\nOur pizza sizes:\n" | randtype -t 0,1000
for p in ${!sizeArray[*]}
do
printf "$(($p +1)). ${sizeArray[$p]}-${priceArray[$p]}\n" | randtype -t 0,1000
done
printf "\nOur signature pizzas:\n" | randtype -t 0,1000
for p in ${!pizzaArray[*]}
do
printf "$(($p +1)). ${pizzaArray[$p]}\n" | randtype -t 0,1000
done

#=================================================Starting off
tput setaf 3
printf "\n\n===============================\n\n" | randtype -l

read -p "Would you like to place an order? " n2

printf "\n\n===============================\n\n"

#-------------Loop for amount of pizzas
while [[ $n2 = "y" ]] || [[ $answer = "y" ]]
do
 printf "\nChoose your size:\n"
PS3="Please select your size:  "
  select size in "${sizeArray[@]}"
   do

	case $size in
	"Extra Large")
		break;;
	"Large")
		break;;
	"Medium")
		break;;
	"Small")
		break;;
		*)
<<<<<<< HEAD
	esac
=======
		printf "Sorry, what was that again?\n\n "
       esac
>>>>>>> 76695d3fe2856ab497a2c352fbe8fda06ffc5335
done
printf "Sorry, what was that again?\n\n"
printf "\nChoose your style:\n"
PS3="Please select the type of pizza:"
     select style in "${pizzaArray[@]}"
         do
        case $style in
        "Cheese")
                break;;
        "Pepperoni")
                break;;
        "Hawaiian")
                break;;
        "Veggie")
                break;;
        "Vegan")
                break;;
        "Meat Lovers")
                break;;
        	'*') printf "Sorry, what was that again?\n"
<<<<<<< HEAD
=======
esac
>>>>>>> 76695d3fe2856ab497a2c352fbe8fda06ffc5335

	esac
done

read -p "Do you want another pizza? (y/n) " answer

if [[ $answer = "n" ]] || [[ $n2 = "n" ]]
then
break
elif
printf "\nPlease select y/n\n"
fi
done



#==================SUMMARY
tput setaf 7
tput bold
printf "==================\nOrder\n==================\n"
for s in ${!tPizzaArray[*]}
do
printf "$(($s+1)). ${size[$s]} - ${style[$s]}\n"
done
printf "=================\nHave a good day!\n=================\n"



PIZZA2.sh
#!/bin/bash

#---------------------TITLE LOGO
clear
tput setaf 6
tput setab 1
tput bold
cat LOGO.txt | randtype -l
sleep 1s
tput sgr0

#------ARRAYS
topArray=("Pepperoni" "Ham" "Pineapples" "Red Peppers" "Green Peppers" "Olives" "Sausage" "Meatballs")
pizzaArray=("Cheese" "Pepperoni" "Hawaiian" "Veggie" "Vegan" "MeatLovers")
sizeArray=("Extra Large" "Large" "Medium" "Small")
priceArray=('$15.99' '$13.99' '$10.99' '$8.99')
#---------Greeting
tput setaf 2
tput bold
printf "\n\nWelcome to PIZZA GENERATION!\n" | randtype -t 0,10000
tput sgr0
sleep 1s
#------------------------------=======----------PIZZA MENU
tput setaf 3
tput bold
printf "\nOur pizza sizes:\n" | randtype -t 0,1000
for s in ${!sizeArray[*]}
do
printf "$(($s +1)). ${sizeArray[$s]}-${priceArray[$s]}\n" | randtype -t 0,1000
done
printf "\nOur signature pizzas:\n" | randtype -t 0,1000
for s in ${!pizzaArray[*]}
do
printf "$(($s +1)). ${pizzaArray[$s]}\n" | randtype -t 0,1000
done

printf '\nChoose your own toppings! $1 each!\n' | randtype -t 0,1000
for s in ${!topArray[*]}
do
printf "$(($s +1)). ${topArray[$s]}\n" | randtype -t 0,1000
done
#=================================================Starting off
tput setaf 3
printf "\n\n===============================\n\n" | randtype -l

read -p "Would you like to place an order? " n2

if [ $n2 = "yes" ] || [ $n2 = "Yes" ] || [ $n2 = "Y" ] || [ $n2 = "y" ]
then
tput bold
tput setaf 3
printf "\n\nLet's start!\n"
sleep 1s
else
tput setaf 1
tput bold
printf "\nSee you around!\n"
sleep 2s
reset
exit;
fi
tput sgr0
tput bold
tput setaf 6
#-------------------------------SELECTION



DPIZZA2ndEDIT.sh
#!/bin/bash

#------------TITLE LOGO
clear
tput setaf 6
tput setab 1
tput bold
cat LOGO.txt | randtype -l
sleep 1s
tput sgr0

#------ARRAYS
pizzaArray=("Cheese" "Pepperoni" "Hawaiian" "Veggie" "Vegan" "MeatLovers")
sizeArray=("Extra Large" "Large" "Medium" "Small")
priceArray=('$15.99' '$13.99' '$10.99' '$8.99')
tPizzaArray=$($PS3)
tSizeArray=()
#---------Greeting
tput setaf 2
tput bold
printf "\n\nWelcome to PIZZA GENERATION!\n" | randtype -t 0,10000
tput sgr0
sleep 1s
#------------------------------=======----------PIZZA MENU
tput setaf 3
tput bold
printf "\nOur pizza sizes:\n" | randtype -t 0,1000
for p in ${!sizeArray[*]}
do
printf "$(($p +1)). ${sizeArray[$p]}-${priceArray[$p]}\n" | randtype -t 0,1000
done
printf "\nOur signature pizzas:\n" | randtype -t 0,1000
for p in ${!pizzaArray[*]}
do
printf "$(($p +1)). ${pizzaArray[$p]}\n" | randtype -t 0,1000
done

#=================================================Starting off
tput setaf 3
printf "\n\n===============================\n\n" | randtype -l
read -p "Would you like to place an order? " n2
printf "\n===============================\n\n"
if [[ $n2 = "y" ]]
then
echo "Let's get started!"
sleep 1s
elif [[ $n2 = "n" ]]
then
echo "So...why are you here?"
exit
elif [[ $n2 = "*" ]]
then
echo "Say that again? "
fi
tput bold
tput setaf 3
#===============================================Loop for amount of pizzas
while [[ $n2 = "y" ]] || [[ $answer = "y" ]] 
do
 printf "\nChoose your size:\n"
PS3="What size Pizza do you want? "
  select size in "${sizeArray[@]}" #select is a loop for options
   do					#case is used to run a command based on pattern matching

	case $size in
	"Extra Large")
		break;;
	"Large")
		break;;
	"Medium")
		break;;
	"Small")
		break;;
	      *)
		printf "\nSorry, what was that again?\n"
	esac
done
printf "\nChoose your style:\n"
PS3="Please select the type of pizza: "
     select style in "${pizzaArray[@]}"
         do
        case $style in
        "Cheese")
                break;;
        "Pepperoni")
                break;;
        "Hawaiian")
                break;;
        "Veggie")
                break;;
        "Vegan")
                break;;
	"MeatLovers")
		break;;

        	*) 
		printf "Sorry, what was that again?\n"

	esac
done
printf "\n "
read -p "Do you want another pizza? (y/n) " answer

if [[ $answer = "n" ]] || [[ $n2 = "n" ]]  
then
break
fi
done

#==================SUMMARY
tput setaf 7
tput bold
printf "==================\nOrder\n==================\n"
for s in ${!tPizzaArray[*]}
do
printf "$(($s+1)). ${size[$s]} - ${style[$s]}\n"
done


PIZZAGEN.sh
#!/bin/bash

#=======================TITLE LOGO
clear
tput setaf 6 #<------Sets font color#
tput setab 1 #<------Sets text background color#
tput bold    #<-----------Sets the text to bold#
cat LOGO.txt | randtype -t 0,1000  #<--------randtype displays line as self typing#
sleep 1s   #<------------Delays next section by 1 second
tput sgr0 #<------------turns off all text formatting after this section#

#==========================================ARRAYS
pizzaArray=("Cheese" "Pepperoni" "Hawaiian" "Veggie" "Vegan" "MeatLovers")
sizeArray=("Extra Large" "Large" "Medium" "Small")
psizeArray=('$15' '$13' '$10' '$8')
pPizzaArray=('$1' '$2' '$3' '$5' '$5' '$3')
tPizzaArray=()
total=0
price=()
#========================================GREETING
tput setaf 4
tput bold
printf "\n\nWelcome to PIZZA GENERATION!\n" | randtype -t 0,10000
tput sgr0
sleep 1s
#======================================CALLING THE PIZZA MENU ARRAYS
tput setaf 3
tput bold
printf "\nOur pizza sizes:\n" | randtype -t 0,1000
for p in ${!sizeArray[*]}
do
printf "$(($p +1)). ${sizeArray[$p]}-${psizeArray[$p]}\n" | randtype -t 0,1000
done
printf "\nOur signature pizzas:\n" | randtype -t 0,1000
for p in ${!pizzaArray[*]}
do
printf "$(($p +1)). ${pizzaArray[$p]}-${pPizzaArray[$p]}\n" | randtype -t 0,1000
done

#==============================================PLACING THE ORDER PROMPT
tput setaf 6

function yesno #------Function for placing order 
{
printf "\n===============================\n"
 read -p "Would you like to place an order? " ntwo
printf "===============================\n"

    case $ntwo in
	[yY]*)
       	;;
        [nN]*)
       echo "See you again!"
       exit;;
      *)
      echo "Please enter y or n!"
yesno;;
esac
}
yesno


tput bold
tput setaf 2
#======================================Select Case Loop for choosing options

while [[ $ntwo = "y" ]] || [[ $ntwo = "Y" ]] 
do
 printf "\nChoose your size:\n"
PS3="What size pizza do you want? "   #PS3(Prompt String 3) is basically a read prompt
  select size in "${sizeArray[@]}"     #<--------------select is a loop for options
   do				       #<----------case is used to run a command based on pattern matching
       case $size in
        "Extra Large")
		price+="15 "
		tPizzaArray+=('Extra Large')
		break;;
	   "Large")
		price+="13 "
		tPizzaArray+=('Large')
		break;;
	   "Medium")
		price+="10 "
		tPizzaArray+=('Medium')
		break;;
	    "Small")
		price+="9 "
		tPizzaArray+=('Small')
		break;;
	      *)
		echo "------------"
		echo "Please select a number from the list!"
		echo "------------"

esac
done
#===============================================================
printf "\nChoose your style:\n"
PS3="Please select the type of pizza: "
     select style in "${pizzaArray[@]}"
         do
        case $style in
        "Cheese")
		price+="1 "
		tSizeArray+=('Cheese')
                break;;
        "Pepperoni")
		price+="2 "
		tSizeArray+=('Pepperoni')
                break;;
        "Hawaiian")
		price+="3 "
		tSizeArray+=('Hawaiian')
                break;;
        "Veggie")
		price+="5 "
		tSizeArray+=('Veggie')
                break;;
        "Vegan")
		price+="5 "
		tSizeArray+=('Vegan')
                break;;
	"MeatLovers")
		price+="3 "
 		tSizeArray+=('MeatLovers')
		break;;

        	*) 
		echo "------------"
		echo "Please select a number from the list!"
		echo "------------"

	esac
done
printf "\n "
#=======================================Function  

function anotherp
{
read -p "Do you want another pizza? (y/n) " nthree
case $nthree in 
   [nN]*)
   echo "Lets continue to your order summary!"
ntwo="n";;
   [yY]*)
	;;
       *)
	echo "-----"
	echo "Press y or n!"
	echo "----"
anotherp;;

esac
}


anotherp
#tput bold
#tput setaf 2
done
#=============================================SUMMARY
tput setaf 7
tput bold
printf "=======================\nOrder Summary\n=======================\n"
for s in ${!tPizzaArray[*]}
   do
	printf "$(($s+1)). ${tPizzaArray[$s]} - ${tSizeArray[$s]}\n"
done
	printf "=======================\nTOTAL\n=======================\n"
for n in ${price[@]}
  do
 	(( total += n ))
 done
#============================TAX FUNCTION
function tax
{
T=.07
TAX=$( echo "$total*$T"|bc)
COMPLETE=$(echo "$total+$TAX"|bc)
   (echo "---------Total:" '$'"$COMPLETE")
}
#============================TOTAL
echo "Your subtotal is:" '$'"$total"
echo "-------------Tax: %7"
tax
echo ""
read -p "<Press enter to exit> " 
reset 
